news/2020-02-03-monthly-beatmapping-contest-february.md

```diff
@@ -51,39 +50,39 @@ This month's artist is **Voicians**, and the following songs are viable for cont
 
 #### Closer (172 BPM)
 
-Beatmap template download: [Full version](https://assets.ppy.sh/artists/59/osz/Voicians%20-%20Closer.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/89/files/Voicians_-_Closer.osz) (2:20)
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/59/osz/Voicians%20-%20Closer.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/89/files/Voicians_-_Closer.osz?2020-1) (2:20)
```

Comment:
why was this necessary? there doesn't seem to be any caching issue

Reply:
I think it's more for versioning, making sure you get the right version of the osz.

Reply:
was told to add it by guy who uploaded files to assets.ppy.sh

Reply:
well it doesn't really harm anything either so good enough for me.